sdk/loadtestservice/azure-mgmt-loadtestservice/dev_requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e ../../../tools/azure-devtools
+-e ../../../tools/azure-sdk-tools
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/25264/files#r928362974